src/library/driftory.ts

```diff
@@ -329,7 +326,8 @@ export default class Driftory {
     }
   }
 
-  /** Get the currently active frame index (if there is one) */
+  /** Get the currently active frame index. This will be whatever frame is in the middle of the
```

Comment:
Thanks for the extra explanation. I'm glad it also detects mouse drag 😊

Reply:
Awesome. Thank you for getting the documentation started… Seeing how you perceive things helps me understand what needs expanding. Together we can documented better than either of us could on our own.